addons/languages/slovak/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```

Comment:
I assume this pack is the published pack at https://play.google.com/store/apps/details?id=org.herrlado.ask.languagepack.slovak

If so, you need to also override the application ID to match `org.herrlado.ask.languagepack.slovak`

Reply:
Is it OK now?

Reply:
@menny, is it OK now?

Reply:
Yes. Thanks